obstacleInitTrust: 0.8
obstacleWidth: 0.3
minBlobElements: 10
maxBlobElements: 300
blobMaxDistance: 0.1
